CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,23 @@
+Spacetelescope Code of Conduct
```

Comment:
This should probably be a header, which in `md` means it should be ``# Spacetelescope Code of Conduct``.

Maybe also the words should be ``# Spacetelescope Github Code of Conduct`` or ``# Spacetelescope Open Source Code of Conduct``?  Currently it seems like the CoC for *all* of STScI, which is probably going too far...

Reply:
good suggestions, thanks.